src/main/java/com/flippingutilities/ui/flipping/FlippingPanel.java

```diff
@@ -279,10 +221,10 @@ public void rebuild(List<FlippingItem> flippingItems)
 					return item1.getLatestActivityTime().compareTo(item2.getLatestActivityTime());
 				});
 				break;
-			case "favorite":
+			case FAVORITE:
 				result = result.stream().filter(item -> item.isFavorite()).collect(Collectors.toList());
 				break;
-			case "profit":
+			case PROFIT:
```

Comment:
leaving this stuff in as I'll be adding them back later when I figure out a better UX/UI  for including them